resource_booking/i18n/es.po

```diff
@@ -33,19 +31,19 @@ msgstr ""
 #: code:addons/resource_booking/models/resource_booking.py:0
 #, python-format
 msgid "%(partner)s - %(type)s"
-msgstr "%(partner)s - %(type)s"
+msgstr ""
 
 #. module: resource_booking
 #: code:addons/resource_booking/models/resource_booking.py:0
 #, python-format
 msgid "%(partner)s - %(type)s - %(time)s"
-msgstr "%(partner)s - %(type)s - %(time)s"
+msgstr ""
```

Comment:
Don't empty these translations. Anyway, you can add t-translate="off" on the XML for not making this translatable, as it's not required.